wafflehaus/nova/networking/network_count_check.py

```diff
@@ -119,7 +119,7 @@ def _get_networks(body):
         networks = body.get("networks")
         if networks is None:
             return None
-        return [n["uuid"] for n in networks]
+        return [n["uuid"] for n in networks if "uuid" in n]
```

Comment:
I'll have to look into how the port uuid can be resolved to a network uuid, I'm not sure how feasible it is to do from here.  I'm mainly concerned with fixing the AttributeError which leads to 500s in the Nova API when this is used, but bypassing limits isn't good.  What would be a graceful failure mode for this code?